public/buttonComponent.js

```diff
@@ -0,0 +1,45 @@
+export class ButtonComponent extends HTMLElement {
+    constructor() {
+        super();
+        this.selectButtons =[];
+        this.feedbackButton=document.createElement("button");
+    }
+    connectedCallback() {
```

Comment:
```suggestion
 
    connectedCallback() {
```

関数のブロック前後には空行が合ったほうが読みやすい気がします。